ui/src/store/configureStore.ts

```diff
@@ -73,6 +74,7 @@ export const rootReducer = combineReducers<ReducerState>({
   }),
   currentPage: currentPageReducer,
   currentDashboard: currentDashboardReducer,
+  data: dataReducer,
```

Comment:
maybe call this `queryCache` and `queryCacheReducer`?. data can mean sooo many things... :/

Reply:
totally. Just seeing this. how about dashboardQueryCache? or are we thinking something a little less specific?

Reply:
I like queryCache, I can't tell if we will only use this in dashboards or not.